UnitySDK/.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Can you add the .vscode folder to .gitignore?

Reply:
I'll add it. Should I remove that folder from these changes too?

Reply:
Once you update it they won't be tracked to git. You should do that before you merge